spec/support/capybara.rb

```diff
@@ -4,9 +4,13 @@
 
 Capybara.javascript_driver = :webkit
 
-# Capybara::Webkit.configure do |config|
-#   config.skip_image_loading
-# end
+Capybara::Webkit.configure do |config|
+  # Silently return an empty 200 response for any requests to unknown URLs.
+  config.block_unknown_urls
+
+  # Timeout if requests take longer than 20 seconds
+  config.timeout = 20
```

Comment:
@yunixon this timeout looks to long for tests. Change it to 5 seconds.

Reply:
@shir with 5 seconds some specs randomly failed

Reply:
@yunixon it's the issue you should fix. No requests should take more then 5 seconds.